lib/watchpack.js

```diff
@@ -184,6 +184,9 @@ class Watchpack extends EventEmitter {
 		const ignored = this.watcherOptions.ignored;
 		const filter = path => !ignored(path);
 		const addToMap = (map, key, item) => {
+			if (key === undefined) {
+				return;
+			}
```

Comment:
I think better to throw error/output warning here

Reply:
The TypeError was already thrown after this and the message is reasonable to understand what's wrong. I don't think we need to add extra error throwing. Then this PR is not the right repository to solve the issue. I will close this PR and send a PR to the client side of the library. Thanks for reviewing.